### DISCUSSION TITLE
Welcome to browser.engineering discussions!

### DISCUSSION THREAD

**Discussion by chrishtr** [browserengineering/book#208] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to talk about the [book](https://browser.engineering).
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by manish-mehra** [comment]
Hello @chrishtr 
As someone who has relied on Browsers as much as his code editor during development for past two years. Recently I found myself wondering, what must be happening under the hood of browsers,  and I came across this book. It looks fascinating and I'm glad something like this exists. Thank you for writing! 

I want to know, what are the prerequisites for following this book other than python, html, css, and javascript? 

Good Day!

**Reply by chrishtr** [reply]
Glad you are interested in the book!

To answer your question about prerequisites: I think there aren't any beyond the ones you mentioned. The book does assume a general knowledge of programming and Python in particular, though if you know another language you can probably learn it as you go.

**Reply by manish-mehra** [reply]
Thank you!!! @chrishtr

**Comment by MetaverseMorgan** [comment]
Just discovered this exists thanks to Robin's newsletter and ordered it immediately. The website is beautiful, safe to say I'll be diving in this weekend, really excited to learn more and talk about it on here.

**Reply by pavpanchekha** [reply]
We love to hear it!
